### PR TITLE
Drag and drop effect for entire column

### PR DESCRIPTION
**Pull request by kevinralph**
I added an optional drag and drop effect for the entire column (as opposed to just the header cell) for the movable columns feature. Set 'enableMovableChildCells' to true in gridOptions to use this feature. Screenshot provided below as a preview of the feature.

screen shot 2016-10-26 at 3 17 26 pm

### CODE REVIEW

**Comment by kevinralph** [comment]
I don't think the failure is related to my changes.

**Comment by mportuga** [comment]
@dlgski This is a change in the design, I am not sure I agree with having the whole column be visible in drag and drop. What do you think? Should we take this change?

@kevinralph What is your reason for wanting this change?

**Comment by kevinralph** [comment]
@mportuga It is a design change but it is optional. You have to set enableMovableChildCells to true in gridOptions to see it.  A client wanted this in a project I was working on and it would have saved me some time if the grid came with this option.  I saw other modules with this included but they lacked in a lot of other areas.  I figured I can't be the only person who's faced this problem and if I can save other people time in the future, why not contribute?

**Comment by mportuga** [comment]
Thanks for the reply and contributing @kevinralph! I hadn't noticed the feature toggle, but as long as that is the case, I am okay with it, I will approve and someone else from the UI-grid team should merge it after they review it.

File: src/features/move-columns/less/colMovable.less [hunk 1]
```diff
@@ -6,6 +6,60 @@
...
+      border-bottom: 1px solid #d4d4d4;
+    }
+    .ui-grid-cell {
+      display: block;
+      visibility: visible;
+      cursor: grabbing;
+      cursor: -moz-grabbing;
+      cursor: -webkit-grabbing;
+      font-weight: normal;
+
```
Comment (mportuga) [inline src/features/move-columns/less/colMovable.less]: You have a bit of unnecessary extra padding here.

**Comment by dlgski** [comment]
I would like to merge this, but it is a new feature and we need documentation on it

**Comment by kevinralph** [comment]
@dlgski Ok, great.  I'll try to get some documentation in sometime this week.

**Comment by dlgski** [comment]
still waiting on documentation

**Comment by kevinralph** [comment]
@dlgski Sorry about the wait. I've been busy with work. I'll try to get something in soon.

**Comment by mportuga** [comment]
Closing due to inactivity. Feel free to reopen when you have added the documentation.

**Pull request closed without merging**